src/treeinterp/OffensiveStrategy.cpp

```diff
@@ -0,0 +1,85 @@
+//
+// Created by jesse on 14.10.19.
+//
+#include "bt/BehaviorTree.hpp"
+#include "bt/decorators/Repeater.hpp"
+#include "treeinterp/OffensiveStrategy.h"
+#include "bt/tactics/DefaultTactic.h"
+#include "bt/Role.h"
+#include "skills/gotopos/GoToPos.h"
+#include "skills/Attack.h"
+
+#include "include/roboteam_ai/treeinterp/CoachDefenderRole.h"
+
+
+namespace bt {
+
+    OffensiveStrategy::OffensiveStrategy() {
+        // Set the robottypes for the robot so the robotdealer can decide which robot should do what
+        this->robots = {
```

Comment:
you can do this in the class definition already

std::map... Robots = {
   { "o1", ... },
   ...
} 

Allows for more optimizations + a default ctor